core/storage/user/gae_models.py

```diff
@@ -288,6 +326,25 @@ def get(cls, user_id, exploration_id):
         return super(ExpUserLastPlaythroughModel, cls).get(
             instance_id, strict=False)
 
+    @classmethod
+    def export_data(cls, user_id):
+        """Takeout: Export ExpUserLastPlaythroughModel user-relevant properties.
+        Args:
+            user_id: str. The user_id denotes which user's data to extract.
+        Returns:
+            dict. The user-relevant properties of ExpUserLastPlaythroughModel
+            in a python dict format.
+        """
+        found_models = cls.get_all().filter(cls.user_id == user_id)
+        user_data = {}
+        for user_model in found_models:
+            user_data[user_model.exploration_id] = {
+                'exp_version': user_model.last_played_exp_version,
+                'card_title': user_model.last_played_state_name
```

Comment:
Actually, on second thoughts, let's call the key state_name (please update the doc as well). This will help with consistency in terminology, since we refer to this as "state" everywhere else.

Reply:
Ack